packages/organizations/reducer.js

```diff
@@ -0,0 +1,26 @@
+const orgData =
+  typeof window !== 'undefined' &&
+  typeof window.bufferData !== 'undefined' &&
+  typeof window.bufferData.organizations !== 'undefined' &&
+  window.bufferData.organizations;
```

Comment:
I think this can be simplified to just be something like this instead 😉 
```suggestion
const orgData = window?.bufferData?.organizations;
```